### ISSUE TITLE
OkHttp outdated

### ISSUE THREAD

**Issue by geertberkers** [ibm-bluemix-mobile-services/bms-clientsdk-android-core#69]
My application cannot fetch https requests because you haven't updated the OkHttp library yet.

You are currently using version 2.7.4.
And this is the only reference in my project to this library.

```
+--- project :pgogateway
|    +--- com.ibm.mobilefirstplatform.clientsdk.android:core:3.0.0
|    |    +--- com.squareup.okhttp:okhttp:2.7.4
|    |    |    \--- com.squareup.okio:okio:1.6.0
```

Error:
```
W/System.err: java.net.ProtocolException: Expected ':status' header not present
        at com.squareup.okhttp.internal.http.Http2xStream.readHttp2HeadersList(Http2xStream.java:267)
W/System.err:     at com.squareup.okhttp.internal.http.Http2xStream.readResponseHeaders(Http2xStream.java:149)
        at com.squareup.okhttp.internal.http.HttpEngine.readNetworkResponse(HttpEngine.java:737)
        at com.squareup.okhttp.internal.http.HttpEngine.access$200(HttpEngine.java:87)
        at com.squareup.okhttp.internal.http.HttpEngine$NetworkInterceptorChain.proceed(HttpEngine.java:722)
        at com.squareup.okhttp.internal.http.HttpEngine.readResponse(HttpEngine.java:576)
        at com.squareup.okhttp.Call.getResponse(Call.java:287)
W/System.err:     at com.squareup.okhttp.Call$ApplicationInterceptorChain.proceed(Call.java:243)
        at com.squareup.okhttp.Call.getResponseWithInterceptorChain(Call.java:205)
        at com.squareup.okhttp.Call.access$100(Call.java:35)
        at com.squareup.okhttp.Call$AsyncCall.execute(Call.java:171)
        at com.squareup.okhttp.internal.NamedRunnable.run(NamedRunnable.java:33)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
W/System.err:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:764)
```

**Comment by AnanthaKrish** [comment]
@geertberkers Hi, you are using an old version of Core SDK. Please use the version 3.1.6. Let me know if you face any issues.

**Comment by geertberkers** [comment]
Yes the problem is solved in 3.1.6.
I could not find it because I thought 3.0.0 was the latest version (https://github.com/ibm-bluemix-mobile-services/bms-clientsdk-android-core/releases)
